app/cdap/components/shared/DiscardDraftModal/index.tsx

```diff
@@ -34,16 +34,20 @@ export const DiscardDraftModal = ({
   discardFn,
 }: IDiscardDraftModalProps) => {
   return (
-    <ConfirmationModal
-      headerTitle={T.translate(`${PREFIX}.title`)}
-      isOpen={isOpen}
-      cancelFn={discardFn}
-      cancelButtonText={T.translate(`${PREFIX}.discard`)}
-      confirmButtonText={T.translate(`${PREFIX}.continue`)}
-      toggleModal={toggleModal}
-      confirmationElem={T.translate(`${PREFIX}.text`)}
-      confirmFn={continueFn}
-      closeable={true}
-    />
+    <>
+      {isOpen && (
```

Comment:
why does this need to be done? this will render twice?

Reply:
use componentdidupdate works. neat